### DISCUSSION TITLE
Make it possible to use a different Model for each subject

### DISCUSSION THREAD

**Discussion by Naegles** [Extraltodeus/multi-subject-render#7] [Ideas]
Imo it would be very useful to be able to use a different model for each foreground subject, for example if you have dreambooth models trained on different people/celebrities/characters. 

Apologies if this is already possible.

**Comment by Extraltodeus** [comment]
I haven't thought of implementing that feature because the switch is quite slow but I'll see if I can easily put that up after I implement the depth2image feature

**Reply by Naegles** [reply]
The webui does have an option for "Checkpoints to cache in RAM" which makes switching between them fairly fast (1-2 seconds for me).

**Reply by Extraltodeus** [reply]
oh that's the use for that option!

**Comment by jakamacca** [comment]
I'll echo this request but also add in a request for changing hypernetworks as well, if not between the foreground subjects then at least between the background and foreground. There's some good hypernetworks out there that really stylize the character but at the cost of oversimplifying the background, it would be good to be able to separate the hypernetworks from each other so we can get the best of both.

**Comment by CCpt5** [comment]
Was this feature ever added?  I have 100+ DB ckpt models and I'm looking to see if there's an extension that would easily allow me to use trained models from 2 different ones in the same image.
